src/content/api/module-variables.md

```diff
@@ -136,6 +137,13 @@ The internal chunk loading function. Takes two arguments:
 Access to the internal object of all modules.
 
 
+### `__webpack_get_script_filename__` (webpack-specific)
+
+This is a function that, when called with a chunk id, returns the filename of that chunk.
+
+T> It is assignable, which allows changing the filename used by the runtime for chunk loading.
```

Comment:
Maybe give an example instead? Reading this, i think it is not making it easier for the readers

Reply:
Yes, please